src/main/java/org/gridsuite/filter/server/dto/IdsByGroup.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.filter.server.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * @author Ghazwa Rehili <ghazwa.rehili at rte-france.com>
+ */
+@Data
+@Builder
+public class IdsByGroup {
```

Comment:
Maybe it's not necessary to define this type ?

Reply:
Map in a request param with Spring MVC has a special meaning.
See class documentation of `RequestParam`

Reply:
Actually it works quite well as we expect for Map<string, string> or MultiValueMap<string, string> and @RequestParam, but yes with this it does not properly deserialize to UUIDs... so ok for the class.